pkg/conduit/runtime.go

```diff
@@ -48,12 +48,12 @@ import (
 	"github.com/conduitio/conduit/pkg/web/api"
 	"github.com/conduitio/conduit/pkg/web/openapi"
 	"github.com/conduitio/conduit/pkg/web/ui"
+	apiv1 "github.com/conduitio/conduit/proto/gen/api/v1"
 	grpcruntime "github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/piotrkowalczuk/promgrpc/v4"
 	promclient "github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/rs/zerolog"
-	apiv1 "go.buf.build/conduitio/conduit/conduitio/conduit/api/v1"
```

Comment:
Our initial plan was to keep the same imports and point them to the code in `proto/gen` using `go mod replace`. However, golangci-lint was complaining about the `replace` directive, and it wasn't possible to ignore just that line.

Reply:
Oh well 🤷 We can replace back again when/if we move back to remote generation.